fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java

```diff
@@ -371,11 +395,10 @@ public Type mysqlTypeToDoris(JdbcFieldSchema fieldSchema) {
             case "BIGINT":
                 return Type.BIGINT;
             case "DATE":
-                return ScalarType.getDefaultDateType(Type.DATE);
+                return Type.DATE;
```

Comment:
Why not still using `ScalarType.getDefaultDateType`, so that we can control the behavior by FE config?

Reply:
Because jdbc external table do not support convert other database type to DATEV2/TIMESTAMPV2 type of doris